wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/RuntimeInfrastructure.java

```diff
@@ -37,31 +26,16 @@
  * @author gazarenkov
  */
 public abstract class RuntimeInfrastructure {
-
   private final Set<String> recipeTypes;
   private final String name;
-  private final InstallerRegistry installerRegistry;
-  private final RecipeRetriever recipeRetriever;
   private final EventService eventService;
-  private final Set<InternalEnvironmentProvisioner> internalEnvironmentProvisioners;
-  private final ServersNormalizer serversNormalizer;
+  //  private final Map<String, InternalEnvironmentFactory> envBuilders = new HashMap<>();
 
-  public RuntimeInfrastructure(
-      String name,
-      Collection<String> types,
-      EventService eventService,
-      InstallerRegistry installerRegistry,
-      RecipeRetriever recipeRetriever,
-      Set<InternalEnvironmentProvisioner> internalEnvironmentProvisioners,
-      ServersNormalizer serversNormalizer) {
-    this.serversNormalizer = serversNormalizer;
+  public RuntimeInfrastructure(String name, Collection<String> types, EventService eventService) {
```

Comment:
`eventService` is unused. Can we remove it?

Reply:
Let's discuss it separately, not related to this PR